docs/getting-started-guides/photon-controller.md

```diff
@@ -0,0 +1,227 @@
+---
+---
+
+The example below creates a Kubernetes cluster using VMware's Photon
+Controller. The cluster will have one Kubernetes master and three
+Kubernetes nodes.
+
+* TOC
+{:toc}
+
+### Prerequisites
+
+1. You need administrator access to a [VMware Photon
+Controller](http://vmware.github.io/photon-controller/)
+deployment. (Administrator access is only required for the initial
+setup: the actual creation of the cluster can be done by anyone.)
+
+2. The [Photon Controller CLI](https://github.com/vmware/photon-controller-cli)
+needs to be installed on the machine on which you'll be running kube-up. If you
+have go installed, this can be easily installed with:
+
+```shell
+go get github.com/vmware/photon-controller-cli
```

Comment:
Oops, my bad. It should be `go get github.com/vmware/photon-controller-cli/photon`. I've updated the pull request to reflect this. Thanks for catching this!